Tasks/UseDotNetV2/globaljsonfetcher.ts

```diff
@@ -0,0 +1,62 @@
+import * as fileSystem from "fs";
+import * as tl from 'azure-pipelines-task-lib/task';
+import { DotNetCoreVersionFetcher } from "./versionfetcher";
+import { VersionInfo } from "./models";
+
+export class globalJsonFetcher {
+
+    private workingDirectory: string;
+    private versionFetcher: DotNetCoreVersionFetcher = new DotNetCoreVersionFetcher();
+    /**
+     * The global json fetcher provider functionality to extract the version information from all global json in the working directory.
+     * @param workingDirectory 
+     */
+    constructor(workingDirectory: string) {
+        this.workingDirectory = workingDirectory;
+    }
+
+    /**
+     * Get all version information from all global.json starting from the working directory.
+     * @param packageType define if you want install the sdk or the runtime.
+     */
+    public async Get(packageType: string): Promise<VersionInfo[]>{
```

Comment:
Method name can be GetVersions given the class name might give the impression that get will get the file list